packages/clay-link/src/__tests__/index.tsx

```diff
@@ -5,7 +5,7 @@
  */
 
 import * as TestRenderer from 'react-test-renderer';
-import ClayLink from '..';
+import ClayLink, {unstable_ClayLinkContext} from '..';
```

Comment:
I think this was lost when sending review... just a tip for when we start doing this more often and have a lot of testing, we can add an alias here for `unstable_ClayLinkContext as ClayLinkContext` and when this is stable we can just change the alias.